docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc

```diff
@@ -7,6 +7,123 @@ The following lists prebuilt rule updates per release. Only rules with
 significant modifications to their query or scope are listed. For detailed
 information about a rule's changes, see the rule's description page.
 
+[float]
+=== 7.12.0
+
+
+This prebuilt rule has been deprecated:
+
+* setgid-bit-set-via-chmod
```

Comment:
This rule, along with the other deprecated one `Execution via Regsvcs/Regasm` are still appearing on the preview link. 

The below should be deleted -  
`docs/detections/prebuilt-rules/rule-details/setuid-setgid-bit-set-via-chmod.asciidoc`
`docs/detections/prebuilt-rules/rule-details/execution-via-regsvcs-regasm.asciidoc`

Reply:
It is really confusing because we had several rules with very similar names/approaches.
`setuid-setgid-bit-set-via-chmod` is still a rule and different from the 7.12 deprecated rule `setgid-bit-set-via-chmod`


`execution-via-regsvcs-regasm` was not removed until after 7.12 and so will reflect in the next doc updates for 7.13

So I think we should be good on both of these